agents/src/worker.ts

```diff
@@ -279,7 +279,7 @@ export class Worker {
   #logger = log().child({ version });
   #inferenceExecutor?: InferenceProcExecutor;
 
-  /* @throws {@link MissingCredentialsError} if URL, API key or API secret are missing */
+  /** @throws MissingCredentialsError if URL, API key or API secret are missing */
```

Comment:
is it correct to remove the links? you should try building the docs locally and seeing if they work